libraries/AP_GPS/AP_GPS.h

```diff
@@ -349,12 +361,14 @@ class AP_GPS
     AP_Int8 _sbas_mode;
     AP_Int8 _min_elevation;
     AP_Int8 _raw_data;
-    AP_Int8 _gnss_mode[2];
-    AP_Int16 _rate_ms[2];
+    AP_Int8 _gnss_mode[GPS_MAX_INSTANCES];
```

Comment:
This is a user facing control parameter, I don't see any value to the blending to be duplicating this.